uavf_2024/imaging/utils.py

```diff
@@ -16,13 +14,52 @@ def calc_match_score(target_desc: TargetDescription, target: Target3D):
     return shape_score * letter_score * shape_color_score * letter_color_score
 
 
-def generate_tiles(img: np.ndarray, tile_size: int) -> "Generator[Tile, None, None]":
-    h, w = img.shape[:2]
-    n_horizontal_tiles = math.ceil(w / tile_size)
-    n_vertical_tiles = math.ceil(h / tile_size)
-    v_indices = np.linspace(0, h - tile_size, n_vertical_tiles).astype(int)
-    h_indices = np.linspace(0, w - tile_size, n_horizontal_tiles).astype(int)
+def generate_tiles(img: np.ndarray, tile_size: int, min_overlap: int = 0) -> "Generator[Tile, None, None]":
```

Comment:
isn't this `min_overlap` parameter not being used anywhere?

Reply:
Lol. This is what happens when you write code in covid delirium.

I just added the parameter and forgot to do anything with it.